css/styles.css

```diff
@@ -497,3 +520,78 @@ textarea {
   text-align: center;
   margin: 1.5rem 0;
 }
+
+/* pop-up styes */
+
+.popup-wrapper {
+  width: 100%;
+  height: 100vh;
+  margin: 0;
+  top: 0;
+  left: 0;
+  background-color: #dfe1e6;
+  position: fixed;
+  z-index: 999;
+  display: none;
+}
+
+.popup {
+  width: 90%;
+  height: 98vh;
+  background: white;
+  padding: 10px;
+  margin: 10px auto;
+  font-family: Arial, Helvetica, sans-serif;
+  position: relative;
+  border-radius: 15px;
+}
```

Comment:
**[REQUIRED CHANGES]**

- Hey, to improve the user experience, Please, kindly adjust your styling for the mobile pop-up. The user is not able to see the  `see live button` and the `see source` button.
_**Every other thing looks great**_ :+1: